tests/text/test_tabstops.py

```diff
@@ -51,6 +56,27 @@ def alignment_get_fixture(self, request):
         return tab_stop, expected_value
 
     @pytest.fixture(params=[
+        ('w:tab',                         'LEFT',
+         'w:tab{w:val=left}'),
+        ('w:tab',                         'RIGHT',
+         'w:tab{w:val=right}'),
+        ('w:tab{w:pos=720}',              'LEFT',
+         'w:tab{w:pos=720,w:val=left}'),
+        ('w:tab{w:pos=720}',              'RIGHT',
+         'w:tab{w:pos=720,w:val=right}'),
+        ('w:tab{w:pos=720,w:val=center}', 'LEFT',
+         'w:tab{w:pos=720,w:val=left}'),
+        ('w:tab{w:pos=720,w:val=center}', 'RIGHT',
+         'w:tab{w:pos=720,w:val=right}')
+    ])
```

Comment:
I cut out a number of these cases, including all the ones that provided a `w:pos` attribute in the "before" XML. In general, the before XML should be the minimum required to demonstrate the behavior. Since the oxml layer doesn't validate on every operation, it works fine without the `w:pos` (or a pre-existing `w:val`, also a required element).